samples/subsys/lorawan/class_a/src/main.c

```diff
@@ -64,9 +63,9 @@ void main(void)
 		.cb = dl_callback
 	};
 
-	lora_dev = device_get_binding(DEFAULT_RADIO);
-	if (!lora_dev) {
-		LOG_ERR("%s Device not found", DEFAULT_RADIO);
+	lora_dev = DEVICE_DT_GET(DEFAULT_RADIO_NODE);
+	if (!device_is_ready(lora_dev)) {
+		printk("%s: device not ready.\n", lora_dev->name);
```

Comment:
Why switching to printk?

Reply:
because I was copying and pasting from other code, will move to LOG_ERR